### DISCUSSION TITLE
How to get notifications

### DISCUSSION THREAD

**Discussion by Adrian-Lock** [MetOffice/simulation-systems#75] [UM]
As a new git user I'm struggling to work out how to get git to send me notification emails when new discussions are added to specific discussion groups (such as this UM "category").  It feels like it must be possible but my initial search has failed.  Anyone got any tips please?

**Comment by andrewcoughtrie** [comment]
Unfortunately the only option is for notification of all discussions, the option is under the watch/custom drop down menu on the repositories front page.
